docs/handbook/concepts.rst

```diff
@@ -25,9 +25,9 @@ Modes
 -----
 
 The ``mode`` of an image defines the type and depth of a pixel in the image.
-Each pixel uses the full range of the bit depth. So a 1-bit pixel has a range
-of 0-1, an 8-bit pixel has a range of 0-255 and so on. The current release
-supports the following standard modes:
+A 1-bit pixel has a range of 0-1, an 8-bit pixel or a 32-bit floating point
+pixel has a range of 0-255, and a 32-bit signed integer has a range of 0-65535.
+These modes are supported:
```

Comment:
Is that range actually correct?  32 bit signed is +-2 billion, that sounds more like 16bit unsigned.

Reply:
According to [numpy docs](https://www.numpy.org/devdocs/user/basics.types.html) 16-bit is 0 - 65535. uint32 is 0 to 4294967295. Ps thanks guys! In desperate need of 16-bit functionality so will be watching this closely.